core/src/main/java/org/elasticsearch/index/query/DisMaxQueryBuilder.java

```diff
@@ -54,6 +61,24 @@ public DisMaxQueryBuilder add(QueryBuilder queryBuilder) {
     }
 
     /**
+     * Add a list of sub-queries to this disjunction.
+     */
+    public DisMaxQueryBuilder add(List<QueryBuilder> queryBuilders) {
```

Comment:
I understand that we dont want to blow up the public API, will see how it looks if I also change this back in BoolQueryBuilder. Will do so with this PR if thats okay, I think it is not in the interface on master. I just remember we added it there together with the readNamedWritableList() to be able to have more concise serialization and parse method. Have a look at the next commit and tell me what you think.